jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
-          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
-      - run: # install and activate virtual environment with pip
-          command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install -r requirements.txt
-      - save_cache: # special step to save dependency cache
-          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+version: 2
+jobs:
+  build:
+    # ...
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.6/site-packages
+      - restore_cache:  # ensure this step occurs *before* installing dependencies
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
+      - run:
+        command: |
+          sudo pip install pipenv
+          pipenv install
+      - save_cache:
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
           paths:
-            - "venv"
-      - run: # run tests
-          command: |
-            . venv/bin/activate
-            python manage.py test
-      - store_artifacts: # special step to store test reports as artifacts
-          path: test-reports/
-          destination: tr1
-      # See https://circleci.com/docs/2.0/deployment-integrations/ for deploy examples    
+            - ".venv"
+            - "/usr/local/bin"
+            - "/usr/local/lib/python3.6/site-packages"
 ```
 
 {% endraw %}
 
-For the complete list of CircleCI configuration keys,
-refer to the [Configuring CircleCI]({{ site.baseurl }}/2.0/configuration-reference/) document.
+**Note:**
+Use the `chown` command
+to grant CircleCI access to dependency locations.
 
-## See Also
-{:.no_toc}
+### Run Tests
 
-- See the [Deploy]({{ site.baseurl }}/2.0/deployment-integrations/) document for example deploy target configurations.
-- See the [Caching Dependencies]({{ site.baseurl }}/2.0/caching/) document for caching strategies.
+Use the `run` step
+to run your test suite.
 
+```yaml
+version: 2
+jobs:
+  build:
+    # ...
+    steps:
+      # ...
+      - run:
+        command: |
+          pipenv run "python manage.py test"
+```
 
+### Upload And Store Test Results
 
+Use the [`store_test_results`]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results) step
+to upload test results to CircleCI.
+These results will display in the **Test Summary** section of the CircleCI application.
+
+Use the [`store_artifacts`]({{ site.baseurl }}/2.0/configuration-reference/#store_artifacts) step
+to save test results as artifacts.
+
+```yaml
+version: 2
+jobs:
+  build:
+    # ...
+    steps:
+      # ...
+      - store_test_results:
+        path: test-results
+      - store_artifacts:
+        path: test-results
+        destination: tr1
+```
+
+### Deploy Application
+
+This Django application is not deployed anywhere.
```

Comment:
This saddens me, and makes our new title feel less truthful.  I would be willing to help figure out a basic flask deployment to include.

Reply:
@eddiewebb it makes me sad, too! However, the original guide also lacked a deployment — do you think we could merge this PR and then break the deploy into a new PR?